### PR TITLE
Updating README.

### PR DESCRIPTION
**Pull request by sfshaza2**
Updating the info in the readme.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,7 +46,8 @@ submodule-cloning techniques:
   `git submodule update --init --remote`
 
 > NOTE: At any time during development you can use the submodule command to
-> refresh submodules: &nbsp;`git pull; git submodule update --init --remote`
+> refresh submodules:<br>
+> &nbsp;`git pull; git submodule update --init --remote`
```
Comment (kwalrath) [inline README.md]: No need for the `&nbsp;`, right? Maybe use ``` instead?

File: README.md [hunk 2]
```diff
@@ -155,102 +159,30 @@ need to manually deploy, use the deploy script and the `default` project:
...
-      print([[strike]]'Hello World'[[/strike]]);
-    }
-    {% endprettify %}
-
-The `prettify` plugin will also unindent your code.
-
-If you want to see how this functionality was added to this site, refer to
-[this commit](https://github.com/flutter/website/commit/ea15f52fe47d3a7b6313ac58d07c66f3b29fe74d).
+Also check out the site-shared
+[wiki](https://github.com/dart-lang/site-shared/wiki)
```
Comment (kwalrath) [inline README.md]: wiki) -> wiki):

Should we say something about moving this stuff to site-shared/doc?
Should we refer to site-shared/doc above?

File: README.md (hunk 1 above)
Comment (sfshaza2) [reply README.md]: Yeah, good pt. I had just decided to leave it, but that's better.

File: README.md [hunk 3]
```diff
@@ -47,7 +47,9 @@ submodule-cloning techniques:
 
 > NOTE: At any time during development you can use the submodule command to
 > refresh submodules:<br>
-> &nbsp;`git pull; git submodule update --init --remote`
+> ```
+> pull; git submodule update --init --remote`
```
Comment (kwalrath) [inline README.md]: The "git" is missing!

**Pull request merged**